test/gui/shared/steps/steps.py

```diff
@@ -87,9 +87,18 @@ def step(context, displayname, host):
     )
 
 
-@Given('user "|any|" has set up a client with default settings and password "|any|"')
-def step(context, username, password):
+def getPasswordForUser(username):
+    userId = username.split()
+    userFirstname = userId[0]
```

Comment:
I dont think this is required
this is equivalent to `userFirstname = username`

Reply:
It is for username which has both first and last name like `Alice Hansen` used here https://github.com/owncloud/client/blob/808284d75bb23a85de3c8068051995dbbf8e18d2/test/gui/tst_addAccount/test.feature#L23

Reply:
That's the display name of the user. the username cannot have spaces I think it would be better to either use only username or only displayname in these kinds of steps.

Reply:
https://github.com/owncloud/client/pull/8656/files#diff-821cad5db62830778c291de069b5182ff9bcdd90951faa2cec9d223e8efbcc20R91 will split the display-name and will only be using the first-name which is also the username